terraform/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to the orb will be documented in this file.
 Orbs are immutable, some orb versions with no significant changes are
 not listed
 
+## ovotech/terraform@1.8.9
+## Added
```

Comment:
Might be worth explicitly putting this as a bugfix, so that it's easier for people who come across the bug in older versions of the orb to find the fixed version?

Reply:
updated in fb9211b